src/coverlet.console/Program.cs

```diff
@@ -29,6 +28,7 @@ static int Main(string[] args)
             CommandOption target = app.Option("-t|--target", "Path to the test runner application.", CommandOptionType.SingleValue);
             CommandOption targs = app.Option("-a|--targetargs", "Arguments to be passed to the test runner.", CommandOptionType.SingleValue);
             CommandOption output = app.Option("-o|--output", "Output of the generated coverage report", CommandOptionType.SingleValue);
+            CommandOption<LogLevel> verbosity = app.Option<LogLevel>("-v|--verbosity", "Sets the verbosity level of the command. Allowed values are q[uiet], m[inimal], n[ormal], d[etailed].", CommandOptionType.SingleValue);
```

Comment:
Never used this, if we specify `q` map first enum starting with `q` so in this case `quiet` or we need to specify full name?

Reply:
I don't know yet. Need to test my change.

Reply:
ok let us know

Reply:
Ok the short forms didn't work. Removed them.